src/Cryptol/TypeCheck/AST.hs

```diff
@@ -267,6 +269,11 @@ instance PP (WithNames Expr) where
                          [ ppW e
                          , hang "where" 2 (vcat (map ppW ds))
                          ]
+      
+      EPropGuards guards _ -> 
```

Comment:
Probably more readable to pretty print as:
```
propguards
  | P => e1
  | Q => e2
```

(i.e., `vcat` instead of `hsep` and indent under the `propguards`)